### PR TITLE
SQOOP-3418: Document decimal support in Hive external import into parquet files

### PR DESCRIPTION
**Pull request by fszabo2**
This is the documentation part of SQOOP-3382 and SQOOP-3396

### CODE REVIEW

File: src/docs/user/import.txt [hunk 1]
```diff
@@ -472,36 +472,48 @@ Enabling Logical Types in Avro and Parquet import for numbers
...
+as NUMBER (20, 5) is stored as is in Oracle, while the equivalent
 DECIMAL (20, 5) is stored as 1.50000 in an SQL server instance.
-This leads to a scale mismatch during avro import.
+This leads to a scale mismatch during the import.
 
-To avoid this error, one can use the sqoop.avro.decimal_padding.enable flag
-to turn on padding with 0s. This flag has to be used together with the
-sqoop.avro.logical_types.decimal.enable flag set to true.
+To avoid this error, one can use the +sqoop.avro.decimal_padding.enable+
+property to turn on padding with 0s during. Naturally, this property is used
```
Comment (szvasas) [inline src/docs/user/import.txt]: during import(?)

File: src/docs/user/import.txt [hunk 2]
```diff
@@ -838,20 +850,27 @@ $ sqoop import --connect jdbc:mysql://db.foo.com/corp \
...
     -Dsqoop.avro.logical_types.decimal.default.precision=38 -Dsqoop.avro.logical_types.decimal.default.scale=10
-    --connect $CON --username $USER --password $PASS --query "select * from table_name where \$CONDITIONS"
+    --connect $MYCONN --username $MYUSER --password $MYPASS --query "select * from table_name where \$CONDITIONS"
     --target-dir hdfs://nameservice1//etl/target_path --as-avrodatafile --verbose -m 1
 
 ----
+
+Enabling logical types in parquet import and also turning on padding with 0s, while specifying default precision and scale as well:
+----
+$ sqoop import -Dsqoop.parquet.decimal_padding.enable=true -Dsqoop.avro.logical_types.decimal.enable=true
```
Comment (szvasas) [inline src/docs/user/import.txt]: I think you wanted to say -Dsqoop.avro.decimal_padding.enable=true and -Dsqoop.parquet.logical_types.decimal.enable=true

**Review by szvasas — COMMENTED** [review]
Hi Feró,

Thank you for improving our documentation! I have left 2 minor comments, please take a look.

File: src/docs/user/import.txt (hunk 1 above)
Comment (fszabo2) [reply src/docs/user/import.txt]: yes!

File: src/docs/user/import.txt (hunk 2 above)
Comment (fszabo2) [reply src/docs/user/import.txt]: Yes!

**Review by szvasas — APPROVED** [review]

**Pull request merged**